### ISSUE TITLE
WCAG 1.3.1,WCAG 3.3.2: Form elements must have labels (#j-search-cpanel-module)

### ISSUE THREAD

**Issue by brianteeman** [joomla-projects/j4adminui#128]
#### Issue
`Form elements must have labels` ([label](https://dequeuniversity.com/rules/axe/3.3/label?application=msftAI))
#### Target application
[Modules (Administrator) - 109 - Administration](http://localhost/kh/administrator/index.php?option=com_modules&view=select&client_id=1)
#### Element path
#j-search-cpanel-module
#### Snippet
`<input type="text" class="form-control" id="j-search-cpanel-module" autocomplete="off" placeholder="Search for modules">`
#### How to fix
    Fix any of the following:
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Form element does not have an implicit (wrapped) <label>
      Form element does not have an explicit <label>
      Element has no title attribute or the title attribute is empty
